### PR TITLE
OpenTelemetry projects proposal

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**

### CODE REVIEW

**Comment by idvoretskyi** [comment]
@SergeyKanzhelev sorry, can you please provide more clarity on this PR?

GSoC 2019 is done, this file is kept in the repo for archival purposes. We currently accept project proposals to the Community Bridge program, please see https://github.com/cncf/soc/blob/master/communitybridge/README.md#how-to-apply

File: 2019.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ List of Participating Projects for 2019:
 - [containerd](#containerd)
 - [Falco](#falco)
 - [Cortex](#cortex)
+- [OpenTelemetry](#opentelemetry)
```
Comment (idvoretskyi) [inline 2019.md]: GSoC 2019 program is closed, there is no need to submit.

**Comment by SergeyKanzhelev** [comment]
yes, this is what I meant... let me read more carefully how to apply =)

**Comment by SergeyKanzhelev** [comment]
So they must go to this document: https://github.com/cncf/soc/blob/master/communitybridge/README.md#how-to-apply itself?

**Comment by idvoretskyi** [comment]
Right, under the section - https://github.com/cncf/soc/blob/master/communitybridge/README.md#project-ideas.

You may find there one sample that was submitted by Kubernetes.

**Pull request closed without merging**